trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py

```diff
@@ -117,13 +189,14 @@ def weissinger_vortex_lattice(conditions,configuration,wing):
     
         Cl = 2*LT/(0.5*Sref)
```

Comment:
While we're here, make this CL and CD. These are 3D

Reply:
Changed. Issue Resolved.